### PR TITLE
BQ: update samples in README for beta 2 changes.

### PR DESCRIPTION
**Pull request by tswast**

### CODE REVIEW

File: bigquery/README.rst [hunk 1]
```diff
@@ -48,39 +48,49 @@ Load data from CSV
...
+    dataset.description = 'my dataset'
+    dataset = client.create_dataset(dataset)  # API request
 
     SCHEMA = [
         SchemaField('full_name', 'STRING', mode='required'),
         SchemaField('age', 'INTEGER', mode='required'),
     ]
-    table = dataset.table('table_name', SCHEMA)
-    table.create()
+    table = dataset_ref.table('table_name')
```
Comment (jba) [inline bigquery/README.rst]: table_ref?

File: bigquery/README.rst [hunk 2]
```diff
@@ -48,39 +48,49 @@ Load data from CSV
...
-    query = client.run_sync_query('%s LIMIT 100' % QUERY)
-    query.timeout_ms = TIMEOUT_MS
-    query.run()
+        'SELECT name FROM `bigquery-public-data.usa_names.usa_1910_2013` '
+        'WHERE state = "TX" '
+        'LIMIT 100')
+    query_job = client.query(QUERY)  # API request
+    rows = query_job.result()  # Waits for query to finish
 
     for row in query.rows:
```
Comment (jba) [inline bigquery/README.rst]: for row in rows?

Comment (tswast) [reply bigquery/README.rst]: Done.

File: bigquery/README.rst (hunk 1 above)
Comment (tswast) [reply bigquery/README.rst]: Yeah, good idea. Done.

This sample is a bit big for my taste, so I also split the dataset creation to its own sample.

**Pull request merged**